### ISSUE TITLE
Custom Streaming quality 

### ISSUE THREAD

**Issue by Akiyoshi06** [Vencord/plugin-requests#495]
### Plugin Description

I wasn't able to find this but, I was wondering if there's a plugin that I can change my VC stream quality, from 1fps 1pixel to streaming 4k 120fps and anything in between. 

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Golonchy** [comment]
i dont think this is an exact match for whaat you are looking for but i think youll find this useful?
https://github.com/philhk/Vencord/tree/better-plugins/src/plugins/betterScreenshare.desktop
this pllugin allows you to further customize your streams specs +it also has a draggable bar if the fixed values
 dont suit you. if you really want the thing you asked for you can customize the plugin's code. (this plugin isn't created by me)
 <img width="519" alt="Screenshot 2024-05-07 at 11 08 21" src="https://github.com/Vencord/plugin-requests/assets/165617563/58e82b08-abda-4313-bad3-7733239aaa42">

**Comment by BeastGitHub17** [comment]
> i dont think this is an exact match for whaat you are looking for but i think youll find this useful?
> 
> https://github.com/philhk/Vencord/tree/better-plugins/src/plugins/betterScreenshare.desktop
> 
> this pllugin allows you to further customize your streams specs +it also has a draggable bar if the fixed values
> 
>  dont suit you. if you really want the thing you asked for you can customize the plugin's code. (this plugin isn't created by me)
> 
>  <img width="519" alt="Screenshot 2024-05-07 at 11 08 21" src="https://github.com/Vencord/plugin-requests/assets/165617563/58e82b08-abda-4313-bad3-7733239aaa42">

How to add this plugin to vencord? I don't see this in the list?

**Comment by mynameisbob1928** [comment]
> How to add this plugin to vencord? I don't see this in the list?

You’ll need to make a custom installation of Vencord and upload it as a user plugin

**Comment by BeastGitHub17** [comment]
> > How to add this plugin to vencord? I don't see this in the list?
> 
> You’ll need to make a custom installation of Vencord and upload it as a user plugin

can you send me a video or tutorial how to do this?

**Comment by mynameisbob1928** [comment]
> can you send me a video or tutorial how to do this?

No but there is a little guide on how to do it on their aim Vencord repo

**Comment by BeastGitHub17** [comment]
> > can you send me a video or tutorial how to do this?
> 
> No but there is a little guide on how to do it on their aim Vencord repo

where can i find that?

**Comment by mynameisbob1928** [comment]
> where can i find that?

[installing](https://github.com/Vendicated/Vencord/blob/main/docs/1_INSTALLING.md)
and [installing custom plugins](https://github.com/Vendicated/Vencord/blob/main/docs/2_PLUGINS.md)

**Comment by AftmostSeeker80** [comment]
> > > How to add this plugin to vencord? I don't see this in the list?
> > 
> > You’ll need to make a custom installation of Vencord and upload it as a user plugin
> 
> can you send me a video or tutorial how to do this?

Quoting venbot on the Discord server:

If you are trying to install plugins from the third party plugins channel, see the pinned message there.
**Do not** ask for further help or you will lose access to support. If you cannot figure it out with the pinned message, it's not for you.

Absolutely no support is provided with such installs. If you want to ask for help with bugs / issues, install the regular way.
